Source/cmake/OptionsWPE.cmake

```
@@ -234,12 +235,13 @@ if (ENABLE_SPEECH_SYNTHESIS)
     endif ()
 endif ()
 
-SET_AND_EXPOSE_TO_BUILD(USE_CAIRO TRUE)
-
 if (USE_CAIRO)
     find_package(Cairo 1.16.0 REQUIRED)
     find_package(Fontconfig 2.13.0 REQUIRED)
     find_package(Freetype 2.9.0 REQUIRED)
+    SET_AND_EXPOSE_TO_BUILD(USE_CAIRO TRUE)
+elseif (USE_SKIA)
+    SET_AND_EXPOSE_TO_BUILD(USE_CAIRO FALSE)
 endif ()
 
```

You have to flip this around, at this point `USE_CAIRO` is not yet defined, only `USE_SKIA`. The options are mutually exclusive so the following will do:

```suggestion
if (USE_SKIA)
    SET_AND_EXPOSE_TO_BUILD(USE_CAIRO FALSE)
else ()
    find_package(Cairo 1.16.0 REQUIRED)
    find_package(Fontconfig 2.13.0 REQUIRED)
    find_package(Freetype 2.9.0 REQUIRED)
    SET_AND_EXPOSE_TO_BUILD(USE_CAIRO TRUE)
endif ()

```

---

Thanks!